### ISSUE TITLE
d.ts file for RoundSlider

### ISSUE THREAD

**Issue by dgroh** [soundar24/roundSlider#40]
Any perspective of creating a TypeDefinition for RoundSlider?

**Comment by soundar24** [comment]
Yes, I already planned to add the typescript definition for roundSlider.. But still i didn't start, i will consider this and include soon...

**Comment by distante** [comment]
I am looking for the TypeDefinition too :)

**Comment by soundar24** [comment]
@distante ya sure, I will start progress on this..
